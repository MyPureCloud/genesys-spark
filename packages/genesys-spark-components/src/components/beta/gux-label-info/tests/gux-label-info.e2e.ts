import { newSparkE2EPage, a11yCheck } from '../../../../test/e2eTestUtils';

describe('gux-label-info-beta', () => {
  describe('#render', () => {
    it(`should render component as expected`, async () => {
      const html = `
        <gux-label-info-beta>
          <span slot="content">This is an information tooltip</span>
        </gux-label-info-beta>
        `;
      const page = await newSparkE2EPage({ html });
      const element = await page.find('gux-label-info-beta');

      await element.hover();
      await a11yCheck(page);

      expect(element.outerHTML).toMatchSnapshot();
    });
  });
});
