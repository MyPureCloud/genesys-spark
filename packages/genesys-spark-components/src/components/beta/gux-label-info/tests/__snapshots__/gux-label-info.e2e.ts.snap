// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gux-label-info-beta #render should render component as expected 1`] = `"<gux-label-info-beta hydrated=""> <span slot="content">This is an information tooltip</span> </gux-label-info-beta>"`;
